www/views/feedback/complete.html

```diff
@@ -7,81 +7,52 @@
     </ion-nav-buttons>
   </ion-nav-bar>
   <ion-content scroll="false">
-    <div ng-show="skipped && isCordova">
-      <div ng-show="score > 3">
-        <div class="title" translate>Invite friends to BitPay!</div>
-        <div class="text-center">
-          <i class="icon addressbook-icon-svg">
-            <img src="img/address-book-add.svg"/>
-          </i>
-        </div>
-        <div class="subtitle">
-          <span translate>Share the love by inviting your friends.</span>
-        </div>
-      </div>
-      <div ng-show="score <= 3">
-        <div class="title" translate>Thank you!</div>
-        <div class="subtitle">
-          <span translate>A member of the team will review your feedback as soon as possible.</span>
-        </div>
-        <div class="subtitle" ng-if="score <= 3 || !isCordova">
-          <span translate>If you have additional feedback, please let us know by tapping the "Send feedback" option in the Settings tab.</span>
-        </div>
-        <div ng-if="score <= 3 || !isCordova">
-          <div class="text-center">
-            <i class="icon icon-svg">
-              <img src="img/illustration-send-feedback.png"/>
-            </i>
+    <a class="close-button" ng-click="close()"><i class="icon ion-ios-close-empty close-home-tip"></i></a>
```

Comment:
This line is duplicated. The back button is working as expected.

Anyway, we could remove the `nav-bar`